### PR TITLE
Like feature

### PR DESCRIPTION
**Pull request by jessicaarrais**

### CODE REVIEW

File: src/components/icon/icon.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.material-icons {
+  margin: 4px;
+}
+
+.md-16 {
```
Comment (davicatunda) [inline src/components/icon/icon.css:5]: md-24 is not needed?

File: src/components/search/Search.tsx [hunk 2]
```diff
@@ -37,20 +52,20 @@ export function Search() {
       <Button
         type="button"
         actionType="secondary"
-        disabled={searchValue.length < 3 ? true : false}
+        disabled={searchValue.length < 2 ? true : false}
```
Comment (davicatunda) [inline src/components/search/Search.tsx:55]: `searchValue.length < 2 ? true : false` is the same as `searchValue.length < 2`

so we can not search just 1 letter?

File: src/pages/recipe-page/RecipePage.tsx [hunk 3]
```diff
@@ -78,6 +87,11 @@ function RecipePage() {
           <p>{instruction.text}</p>
         </div>
       ))}
+      <span className="likes">
+        <Icon icon="favorite" size="md-16" />
+        <p>{data.recipe.likes.toString()}</p>
```
Comment (davicatunda) [inline src/pages/recipe-page/RecipePage.tsx:92]: is the toString needed?

**Review by davicatunda — APPROVED** [review]

File: src/components/search/Search.tsx (hunk 2 above)
Comment (jessicaarrais) [reply src/components/search/Search.tsx:55]: it wasn't making sense. removed.

File: src/pages/recipe-page/RecipePage.tsx (hunk 3 above)
Comment (jessicaarrais) [reply src/pages/recipe-page/RecipePage.tsx:92]: actually, no

**Pull request merged**